community/rds/aurora-serverless/template.yml

```diff
@@ -0,0 +1,70 @@
+---
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Aurora serverless cluster
+
+Parameters:
+  DatabaseName:
+    Type: String
+  MasterUserPassword:
```

Comment:
MasterUserPassword, if read from SSM Parameter Store, should be a SecureString using a dynamic reference: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/dynamic-references.html

Reply:
FYI, this and a few of my other suggestions all come from running cfn-lint against your template:
https://pypi.org/project/cfn-lint/

Reply:
A linter for CF templates? Awesome, thanks :smiley: